.github/workflows/linter-eslint-using-reviewdog.yml

```
@@ -0,0 +1,48 @@
+name: Lint using ESLint + reviewdog
+on:
+  workflow_call:
+    inputs:
+      node-version-file:
+        required: true
```

未指定の場合は`default`を利用したい。

```suggestion
        required: false
```